models/EventDate.ts

```diff
@@ -0,0 +1,38 @@
+import dayjs from 'dayjs';
+import locale_de from 'dayjs/locale/de';
+import duration from 'dayjs/plugin/duration';
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import relativeTime from 'dayjs/plugin/relativeTime';
+import timezone from 'dayjs/plugin/timezone';
+import utc from 'dayjs/plugin/utc';
+
+// DayJS Plugins
+dayjs.extend(localizedFormat);
+dayjs.extend(duration);
+dayjs.extend(relativeTime);
+dayjs.extend(timezone);
+dayjs.extend(utc);
+
+export default interface EventDate {
+    day: number;
+    month: number;
+    year?: number;
+}
+
+export const getDateFromEvent = (eventDate: EventDate): Date => {
+    if (eventDate.year) {
```

Comment:
Isn't something with dayjs also possible here? Looks bit complicated but idk if it's simpler with dayjs

Reply:
Done